src/main/resources/repositories.xml

```diff
@@ -6,11 +6,16 @@ http://www.springframework.org/schema/beans/spring-beans-4.0.xsd
 http://www.springframework.org/schema/util
 http://www.springframework.org/schema/util/spring-util-2.5.xsd">
 
-    <util:list id="messages"></util:list>
+    <util:list id="messages"/>
+    <util:map id="users"/>
```

Comment:
to już Wam nie jest potrzebne. Już nie będziemy nadpisywać, więc spokojnie możecie tworzyć listę w konstruktorze repozytorium.